test/io/recordio.chpl

```diff
@@ -80,17 +80,26 @@ var B: [0..#3] MyRecord;
   reader.close();
 }
 
-// Now read the data. Way 2: provide a readWriteThis method.
+// Now read the data. Way 2: provide readThis/writeThis methods.
 // Note that this didn't work with Chapel 1.11 or earlier.
```

Comment:
I'm not sure this is important information to maintain any more, since it was so long ago

Reply:
Probably not, but since it's already in I'm inclined to leave it be, if it's all the same to you.